packages/aquarius/src/bot/commands/roll.js

```diff
@@ -46,7 +46,7 @@ export default async ({ aquarius, analytics }) => {
 
     const values = Array.from(
       roll.matchAll(
-        /(?:(?<sign>[+-]) )?(?<dieCount>\d+)?d(?<dieType>100|20|12|10|8|6|4)(?: ?(?<type>[+-]) ?(?<modifier>\d+(?!d)))?/gi
+        /(?:(?<sign>[+-]) )?(?<dieCount>\d+)?d(?<dieType>[1-9]\d*)(?: ?(?<type>[+-]) ?(?<modifier>\d+(?!d)))?/gi
```

Comment:
If you want to allow leading zeros, `(?<dieType>[1-9]\d*)` would become `(?<dieType>0*[1-9]\d*)`

Reply:
You'd also want to strip the zeros before emojifying them so probably not worth the effort